src/migrations/20220203152515-bootstrap_accounts.ts

```diff
@@ -0,0 +1,90 @@
+/* eslint @typescript-eslint/no-var-requires: "off" */
+// @ts-expect-error ts complain about "Cannot redeclare block-scoped variable 'crypto'"
+const crypto = require("crypto")
+
+const adminUsers = [
+  {
+    phone: "+16505554327",
+    role: "dealer",
+    needUsdWallet: true,
+  },
+  { phone: "+16505554325", role: "funder" },
+  { phone: "+16505554334", role: "bankowner" },
+]
+
+const getRandomInvalidPhone = () => {
+  return `+abc${Math.floor(Math.random() * 999_999_999_999_999)}`
+}
+
+const defaultWallet = { __v: 0, type: "checking", onchain: [] }
+
+module.exports = {
+  async up(db /*, client */) {
+    for (const userMeta of adminUsers) {
+      const role = userMeta.role
+      let user = await db.collection("users").findOne({ role })
+
+      let userId
+      if (!user) {
+        const phone = getRandomInvalidPhone()
```

Comment:
Just double-checking, what's the thinking behind adding "invalid phone" admin users?

Reply:
phone is a mandatory field (there is an index for it), so we need a phone number. but we don't want anyone to log on this user, so having an invalid phone number is the way to do it. 

I think this will be refactor with the new auth library @bodymindarts

Reply:
Eventually yes - first Kratos needs phone / code auth support (supposedly released in Q2?)